Engine/Makefile-defs.linux

```diff
@@ -1,6 +1,6 @@
 INCDIR = ../Engine ../Common ../Common/libinclude ../Plugins
 LIBDIR =
-CFLAGS := -O2 -g -fsigned-char -Wfatal-errors -DNDEBUG -DAGS_RUNTIME_PATCH_ALLEGRO -DAGS_HAS_CD_AUDIO -DAGS_CASE_SENSITIVE_FILESYSTEM -DALLEGRO_STATICLINK -DLINUX_VERSION -DDISABLE_MPEG_AUDIO -DBUILTIN_PLUGINS -DRTLD_NEXT $(shell pkg-config --cflags freetype2) $(CFLAGS)
+CFLAGS := -O2 -g -fsigned-char -Wfatal-errors -DNDEBUG -DAGS_RUNTIME_PATCH_ALLEGRO -DAGS_HAS_CD_AUDIO -DAGS_CASE_SENSITIVE_FILESYSTEM -D_FILE_OFFSET_BITS=64 -DHAVE_FSEEKO -DALLEGRO_STATICLINK -DLINUX_VERSION -DDISABLE_MPEG_AUDIO -DBUILTIN_PLUGINS -DRTLD_NEXT $(shell pkg-config --cflags freetype2) $(CFLAGS)
 CXXFLAGS := -std=c++11 -fno-rtti -Wno-write-strings $(CXXFLAGS)
```

Comment:
CFLAGS is not used in CXXFLAGS ?

the MacOS makefile below does

Reply:
Hmm.

Makefile.linux has this:
`CXXFLAGS += $(CFLAGS)`

but Makefile.osx has this:
`CXXFLAGS := -std=c++11 -fno-rtti -Wno-write-strings $(CXXFLAGS)

then later -

CXXFLAGS := $(CFLAGS) $(CXXFLAGS)`

which means that in Mac's makefile CFLAGS are duplicated in CXXFLAGS?

Reply:
oh, seems i overlooked the CXXFLAGS += line.